Framework/PythonInterface/mantid/utils/nomad/_median_detector_test.py

```diff
@@ -0,0 +1,298 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
why is this file has `_test` as suffix when it is not performing any kind of test?